plugins/chat/assets/javascripts/discourse/services/chat-drawer-router.js

```diff
@@ -160,6 +160,7 @@ export default class ChatDrawerRouter extends Service {
   @tracked component = null;
   @tracked drawerRoute = null;
   @tracked params = null;
+  @tracked currentRouteName = null;
```

Comment:
It was needed to ensure the plugin outlet is recomputed when the drawer route changes.